src/masonite/foundation/Kernel.py

```diff
@@ -39,6 +39,7 @@ def set_framework_options(self):
         self.application.bind("config.session", "tests.integrations.config.session")
         self.application.bind("config.queue", "tests.integrations.config.queue")
         self.application.bind("config.database", "tests.integrations.config.database")
+        self.application.bind("config.location", "tests/integrations/config")
```

Comment:
How will this be used?

Reply:
For the config the idea is not hard-code the "config" directory.
For example with `masonite-inertia` when I want to publish the config file, I can use `config_path("inertia.py")` to compute the  inertia config output path in the project and it's easier to copy it over. (the user in his project might define an other config directory).

Not the best example as that might be the directory which is less likely to be changed but it's an example

Reply:
gotcha ok tha works